apps/demos/src/side-nav.component.scss

```diff
@@ -0,0 +1,83 @@
+// redeclare variables since the barista build cannot deal
+// with a relative reference to the lib folder here
+$gray-200: #e6e6e6;
```

Comment:
hmm, this makes me a little sad. I am guessing aliased imports also don't work, because it is not an actual node_module, right?
i.e.
```scss
@import "~@dynatrace/barista-components/core/variables";
```

Reply:
Not sure. @lukasholzer can you help out?

Reply:
no we can add it to the includeStyleUrls so then you can import `@import 'variables';`